go/grpc/security/grpc-server-insecure-connection.yaml

```diff
@@ -0,0 +1,16 @@
+rules:
+- id: grpc-server-insecure-connection
+  metadata:
+    owasp: 'A8: Insecure Deserialization'
+    cwe: 'CWE-502: Deserialization of Untrusted Data'
+    references:
+    - https://blog.gopheracademy.com/advent-2019/go-grps-and-tls/#connection-without-encryption
+  message: Found an insecure gRPC connection. This allows for a connection without
+    encryption to this server. A malicious attacker could tamper with the gRPC message,
+    which could compromise the machine.
+  languages:
+  - go
+  severity: ERROR
+  patterns:
+  - pattern-not: $GRPC.NewServer(..., $GRPC.Creds(...), ...)
+  - pattern: $GRPC.NewServer()
```

Comment:
is it strictly for `NewServer()` (without any params passed) or it should work for any except `$GRPC.Creds(...)`
like:
`- pattern: $GRPC.NewServer(...)`

Reply:
> is it strictly for `NewServer()` (without any params passed) or it should work for any except `$GRPC.Creds(...)`
> like:
> `- pattern: $GRPC.NewServer(...)`

Oh, yeah, that's right.  Good catch 👍